pkgs/development/r-modules/generic-builder.nix

```diff
@@ -1,10 +1,14 @@
-{ stdenv, R, xvfb_run, utillinux }:
+{ stdenv, R, tzdata, gettext, gfortran, libcxx, xvfb_run, utillinux }:
 
 { name, buildInputs ? [], ... } @ attrs:
 
 stdenv.mkDerivation ({
-  buildInputs = buildInputs ++ [R] ++
-                stdenv.lib.optionals attrs.requireX [utillinux xvfb_run];
+  buildInputs = buildInputs ++ [ R tzdata gettext gfortran ] ++
```

Comment:
It might have been darwin sandbox related (it requires whitelisting of 
allowed filesystem paths and I don't know if whole /nix/store was 
whitelisted by default), and I cannot really test it now, because the 
sandbox is currently broken :(

Peter Simons wrote:
> In pkgs/development/r-modules/generic-builder.nix
> <https://github.com/NixOS/nixpkgs/pull/10623#discussion_r49007467>:
>
>>
>>   { name, buildInputs ? [], ... } @ attrs:
>>
>>   stdenv.mkDerivation ({
>>  -  buildInputs = buildInputs ++ [R] ++
>>  -                stdenv.lib.optionals attrs.requireX [utillinux xvfb_run];
>>  +  buildInputs = buildInputs ++ [ R tzdata gettext gfortran ] ++
>
> Why are |tzdata|, |gettext|, and |gfortran| added to |buildInputs| in
> the generic builder? Linux works fine without those packages in $PATH
> because records those paths in its internal config files. Does this not
> happpen on Darwin?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/10623/files#r49007467>.
>